Connectors/Autocad/Speckle.Connectors.AutocadShared/HostApp/AutocadDocumentModelStore.cs

```diff
@@ -64,6 +63,10 @@ protected override void LoadState()
     }
 
     string? serializedModelCards = _autocadDocumentManager.ReadModelCards(doc);
+    if (serializedModelCards == null)
+    {
+      return;
```

Comment:
we might need to clear the existing list if we are opening a new empty model, need to test

Reply:
yes this is not correct in Autocad. If it is null we must clear the existing list